json-schema/schema.json

```diff
@@ -160,46 +162,24 @@
     "require_any_field": {
       "$comment": "Please list all fields here so that we can force the existence of one of them in other parts of the schemas.",
       "anyOf": [
-        {"required": ["template:new_field"]},
-        {"required": ["template:xyz"]},
-        {"required": ["template:another_one"]}
+        {"required": ["marble:host_node"]},
+        {"required": ["marble:is_local"]}
       ]
     },
     "fields": {
-      "$comment": "Add your new fields here. Don't require them here, do that above in the corresponding schema.",
       "type": "object",
       "properties": {
-        "template:new_field": {
+        "marble:host_node": {
+          "title": "Marble node hosting this data",
           "type": "string"
         },
-        "template:xyz": {
-          "type": "object",
-          "required": [
-            "x",
-            "y",
-            "z"
-          ],
-          "properties": {
-            "x": {
-              "type": "number"
-            },
-            "y": {
-              "type": "number"
-            },
-            "z": {
-              "type": "number"
-            }
-          }
-        },
-        "template:another_one": {
-          "type": "array",
-          "items": {
-            "type": "number"
-          }
+        "marble:is_local": {
+          "title": "Data locality identifier",
```

Comment:
Are these titles user-facing? If so I don't know if this title is that clear to users... what about:

- Indicates whether the Marble node hosting this data is the same one that provides this catalog

That's wordy ... but something like that

Reply:
I don't know much. It's definitely user facing, but I don't know where. From studying other implementations, I've noticed these are short phrases.